advocacy_docs/partner_docs/VeritasGuide/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Veritas Implementation Guide'
+indexCards: simple
+directoryDefaults:
+  iconName: handshake
+---
+
+<p align="center">
+  <img  src="Images/EDBPartnerProgram.png">
+</p>
+<h1 style="text-align: center;">EDB GlobalConnect Technology Partner Implementation Guide</h1>
+<h3 style="text-align: center;">NetBackup for PostgreSQL</h3>
```

Comment:
Should have vendor name and product.  See other guides as example if needed.
Veritas NetBakup ....